### PR TITLE
Aligned the connect with us tabs

### LINKED ISSUE #1034: The connect with us tabs are not properly aligned
**I'm submitting a ...**  (check one with "x")
- [X] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
The tabs are not properly aligned.
**Expected behavior:**
The icons and the tabs should be properly aligned. 
**Screenshot:**
screen shot 2018-02-16 at 10 13 51 pm

I'm working on this.

### PR DESCRIPTION
**Pull request by harsh-jindal**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Aligned the connect with us tabs.
Before:-
screen shot 2018-03-05 at 10 00 48 pm
After:-
screen shot 2018-03-05 at 10 00 54 pm

Fixes #1034

### CODE REVIEW

File: app/templates/components/footer-main.hbs [hunk 1]
```diff
@@ -46,32 +46,32 @@
         <strong class="item">{{t 'Connect with us'}}</strong>
         {{#if socialLinks.supportUrl}}
           <a class="item" href="{{socialLinks.supportUrl}}" target="_blank" rel="noopener noreferrer">
-            <i class="info icon"></i> {{t 'Support'}}
+            <i class="info icon" style="float:left;margin-left:55px;"></i><span class="align-text">Support</span>
```
Comment (sumedh123) [inline app/templates/components/footer-main.hbs]: Please avoid inline styling. You should do it in stylesheets that are defined in the project.

File: app/templates/components/footer-main.hbs [hunk 2]
```diff
@@ -46,32 +46,32 @@
...
           </a>
         {{/if}}
         {{#if socialLinks.facebookUrl}}
           <a class="item" href="{{socialLinks.facebookUrl}}" target="_blank" rel="noopener noreferrer">
-            <i class="facebook f icon"></i> {{t 'Facebook'}}
+            <i class="facebook f icon" style="float:left;margin-left:55px;"></i><span class="align-text">Facebook</span>
```
Comment (sumedh123) [inline app/templates/components/footer-main.hbs]: lot of redundancy there.

**Comment by sumedh123** [comment]
@harsh-jindal You should use semantic classes as long as possible. If you need custom styling, do it in the stylesheets present in the project. I see there is a lot of redundant code there due to inline styling.

**Comment by niranjan94** [comment]
Deployment link: https://pr-1055-open-event-frontend-fossasia.surge.sh

**Comment by harsh-jindal** [comment]
@sumedh123 I'll change it.

**Comment by sumedh123** [comment]
@geekyd Can you take a look at deployment? I think now it looks weird since only the connect with us tabs are aligned. What are your opinions?

**Comment by harsh-jindal** [comment]
@sumedh123 @niranjan94 I have made the changes.
The correct deployment link is http://align.surge.sh/

**Comment by niranjan94** [comment]
Deployment link: https://pr-1055-open-event-frontend-fossasia.surge.sh

**Comment by CosmicCoder96** [comment]
In fact I agree with what @sumedh123 said, it looks weird since the rest of the footer columns are not left aligned, let's leave it the way it is :)

**Comment by harsh-jindal** [comment]
Okay

**Pull request closed without merging**